### ISSUE TITLE
Strange values after first start-up

### ISSUE THREAD

**Issue by Kuma1337** [sn0wyQ/SQ-Project-CSGO-Arduino#13]
im using arduino ide for upload code, but when i verify its getting error

```
In file included from E:\Arduino\csgo_arduino\csgo_arduino.ino:7:0:
E:\Arduino\csgo_arduino\Helpers\global.h:9:10: fatal error: Helpers/definitions.h: No such file or directory
 #include "Helpers/definitions.h"
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
exit status 1
```
the directory is right but i dont know how to config arduino ide setting

**Comment by sn0wyQ** [comment]
Unfortunately it is Arduino IDE issue. You can use "../Helpers/definitions.h" instead or move *definitions.h* to the directory where csgo_arduino.ino is located

**Comment by Kuma1337** [comment]
thx its working perfect now, great work

**Comment by Kuma1337** [comment]
i got a new problem, when i running pc.exe its just keep search device and not responce to next step.
the port is already set to my local port

**Comment by sn0wyQ** [comment]
Before running PC.exe Arduino must be connected to PC. You need to set port only when loading firmware to Arduino, PC.exe can find port with Arduino automaticly. 

Do you use Arduino Leonardo or something else?

**Comment by Kuma1337** [comment]
yep im using arduino leonardo and install driver just now
![6O@7KRE5TYQQIEJMHMAS9@C](https://user-images.githubusercontent.com/88535107/128519490-674f6b05-e37f-4099-8625-522d0ced8535.png)
![5O1GETKX5QFHEPPG@ C{RCH](https://user-images.githubusercontent.com/88535107/128519501-8ea0d62d-0cc8-42a1-b443-69dae488fefd.png)
--
nvm its actually working now

**Comment by Kuma1337** [comment]
sorry bro borther you again, i complite everything but the aimbot not working and other feature also not working, still have no idea now, its arduino.cpp cant read default setting or something wrong?
![Z6MYIJ}VE7S M LAU75DW_K](https://user-images.githubusercontent.com/88535107/128526599-c3f7001b-ca73-4143-8684-9f3d00dac64e.png)

**Comment by sn0wyQ** [comment]
Oooofff, most likely it is because EEPROM of your Arduino has some data in place of where variables are located before first start... I will fix it when I'll be home.

Right now you can load following .ino file to your Arduino board, reset Arduino and then load csgo_arduino.ino to your board again:
```
#include <Arduino.h>
#include <EEPROM.h>

void setup() {
  for (int i = 0; i < 32; ++i) {
    EEPROM.write(i, 0);
  }
}

void loop() {}
```
This should fix it for now

**Comment by Kuma1337** [comment]
yep its fixed now, i'm working on my lcd issue, it doesnt show anything and feature not working, maybe you can give some ideas

**Comment by BurakBalamir** [comment]
Select button not select and other stuffs not working u have any idea!
![image](https://user-images.githubusercontent.com/31852918/128543905-8c882208-befd-4bc8-a895-9507b07b1c01.jpeg)

**Comment by sn0wyQ** [comment]
> yep its fixed now, i'm working on my lcd issue, it doesnt show anything and feature not working, maybe you can give some ideas

As written in Getting Started:
**Note that you may be required to change pin numbers to fit LiquidCrystal(). So if needed change their definitions in Arduino/Helpers/definitions.h (lines 5-10).**

**Comment by facukoren** [comment]
> > yep its fixed now, i'm working on my lcd issue, it doesnt show anything and feature not working, maybe you can give some ideas
> 
> As written in Getting Started:
> **Note that you may be required to change pin numbers to fit LiquidCrystal(). So if needed change their definitions in Arduino/Helpers/definitions.h (lines 5-10).**

how can i make it to fit?? do u have a video about it?

**Comment by Suspectxyzz** [comment]
I did exactly that, but no function works aim bhop etc (offsets are update )


![image](https://user-images.githubusercontent.com/45514952/173184977-82acde5d-e681-445c-9c20-d06022e93ce6.png)
![image](https://user-images.githubusercontent.com/45514952/173185150-b5a58d07-d7b7-4b3f-b66c-48eec0d23d45.png)
![image](https://user-images.githubusercontent.com/45514952/173185172-00595c01-2f3c-43a6-809b-7c3998d99631.png)

**Comment by listerioa** [comment]
some example of how to change the definitions in Arduino/Helpers/definitions.h (lines 5-10).
